### DISCUSSION TITLE
Why is TouchableOpacity active opacity is 0.2 by default?

### DISCUSSION THREAD

**Discussion by anishamalde** [react-native-tvos/react-native-tvos#636] [General Discussions]
So I understand that this is to maintain parity with Core React Native but keeping the opacity this low by default means most of the time, when focused, the TouchableOpacity element is unreadable and violates accessibility contrast criteria unless the style is overridden. Just looking to understand if there is a reason for this beyond parity?

![Screenshot 2024-01-10 at 15 50 46](https://github.com/react-native-tvos/react-native-tvos/assets/39306477/21348ffa-2cce-46f0-b783-4a8b87f3b649)

**Comment by douglowder** [comment]
We prefer to have parity for values like this, where a change would also affect apps built for mobile devices. This repo supports building apps for both mobile and TV devices.
